### ISSUE TITLE
Default to bench?

### ISSUE THREAD

**Issue by DaniPopes** [CodSpeedHQ/codspeed-rust#60]
`cargo codspeed build` defaults to `--profile release`, vs `cargo bench`/`cargo criterion` defaulting to `--profile bench`. Bench inherits release by default but it can be overridden for the specific purpose of adding more debug info to it vs a normal release build. Is this intentional? Thanks

**Comment by art049** [comment]
Yes, this is intentional. The `bench` profile comes with undocumented limitations regarding custom build processes. 
Completely agree this should be the default behavior when this is possible.
(I don't remember the details of the experiment showing the limitations, though).

Did passing --profile bench work for you?

**Comment by DaniPopes** [comment]
Yes, --profile bench works

**Comment by art049** [comment]
Okay then definitely it would make sense to include this in the next major
